staging/src/k8s.io/legacy-cloud-providers/gce/gce_disks.go

```diff
@@ -990,3 +1024,21 @@ func isGCEError(err error, reason string) bool {
 	}
 	return false
 }
+
+// verifyDisksAttachedToNode takes in an slice of disks that should be attached to an instance, and the
+// slice of disks actually attached to it. It returns a map verifying if the disks are actually attached.
+func verifyDisksAttachedToNode(disksToVerify []string, disksActuallyAttached []*compute.AttachedDisk) map[string]bool {
+	verifiedDisks := make(map[string]bool)
+	diskNamesActuallyAttached := []string{}
+	for _, disk := range disksActuallyAttached {
+		diskNamesActuallyAttached = append(diskNamesActuallyAttached, disk.DeviceName)
```

Comment:
I don't get it - why we create a slice first. Should be:
```
diskNameActuallyAttached := sets.NewString()
for _, disk := range disksActuallyAttached {
  disksNamesActuallyAttached.Insrt(disk.DeviceName)
}
```

Reply:
ack